src/sentry/static/sentry/app/views/organizationIntegrations/integrationDetailedView.tsx

```diff
@@ -136,8 +136,17 @@ class IntegrationDetailedView extends AbstractIntegrationDetailedView<
 
   onDisable = (integration: Integration) => {
     let url: string;
-    const [domainName, orgName] = integration.domainName.split('/');
 
+    if (integration.provider.key === 'vercel') {
+      // kind of a hack since this isn't what the url was stored for
+      // but it's exactly what we need and contains the configuration id
+      // e.g. https://vercel.com/dashboard/integrations/icfg_ySlF4UDnHcIPrAAXjGEiwtxo
+      url = integration.configOrganization[0].nextButton.url;
```

Comment:
@MeredithAnya My suggestion would be to find the `configOrganization` of type `project_mapper` in case we add more options later. Then type cast. So something like:
```
const field: ProjectMapperType | undefined = integration.configOrganization.find( config => config.type === 'project_mapper')
if (field) {
     url = field.nextButton.url;
```